lib/vanagon/platform/windows.rb

```diff
@@ -281,11 +282,11 @@ def add_repository(definition)
         commands = []
 
         if definition.scheme =~ /^(http|ftp|file)/
-          if File.extname(definition.path) == '.ps1'
-            commands << %(powershell.exe -NoProfile -ExecutionPolicy Bypass -Command 'iex ((new-object net.webclient).DownloadString(\"#{definition}\"))')
-          else
-            commands << %(C:/ProgramData/chocolatey/bin/choco.exe source add -n #{definition.host}-#{definition.path.gsub('/', '-')} -s "#{definition}" --debug || echo "Oops, it seems that you don't have chocolatey installed on this system. Please ensure it's there by adding something like 'plat.add_repository 'https://chocolatey.org/install.ps1'' to your platform definition.")
-          end
+          commands << if File.extname(definition.path) == '.ps1'
```

Comment:
I ended up just disabling this cop, as it was throwing too many errors to be useful and enforce the style we want